.circleci/config.yml

```diff
@@ -70,8 +70,7 @@ commands:
         type: string
       consul-k8s-image:
         type: string
-        #default: "docker.mirror.hashicorp.services/hashicorpdev/consul-k8s-control-plane:latest"
-        default: "kyleschochenmaier/consul-k8s-acls"
+        default: "docker.mirror.hashicorp.services/hashicorpdev/consul-k8s-control-plane:$(git rev-parse --short HEAD)"
```

Comment:
This will allow us to use whatever the image is for this commit instead of `latest`. This can go into the `main` branch as it will allow us to not have to set the image for features we're working on.

Reply:
hooray, thanks for doing this, I was trying to figure out how it was possible.
🥇